src/main/java/edu/ie3/datamodel/io/naming/FileNamingStrategy.java

```diff
@@ -311,8 +265,7 @@ public String getIdCoordinateEntityName() {
   public Optional<String> getIdCoordinateFilePath() {
     // do not adapt orElseGet, see https://www.baeldung.com/java-optional-or-else-vs-or-else-get for
     // details
-    return getFilePath(
-        getIdCoordinateEntityName(), fileHierarchy.getBaseDirectory().orElseGet(() -> ""));
+    return getFilePath(getIdCoordinateEntityName(), fileHierarchy.getBaseDirectory().orElse(""));
```

Comment:
As for the comment above this shouldn't be changed. Any reason you did do it nevertheless?